TROUBLESHOOTING.md

```diff
@@ -44,3 +44,10 @@ on GNU Linux, it may be something like
 
 ### Grid-runner
 - When using grid-runner to run grid, individual services have to sometimes be restarted when switching branches
+
+### Accepting certificates
+- You have to manually accept the certificates for the urls for the different grid services.
```

Comment:
I think it is useful to mention it here. Because grid has so many services, this needs to be done multiple times and because of the number of errors, the original problem can be difficult to find even if you are aware of the issue.